src/eventTimingService.ts

```diff
@@ -0,0 +1,27 @@
+import { Dictionary } from '@mparticle/web-sdk';
+
+export interface IEventTimingService {
+    setEventTiming: (eventName: string, timestamp: number) => void;
```

Comment:
It's a string, but it has to be certain strings, so this would be helpful:

```suggestion
    setEventTiming: (eventName: EventTimingName, timestamp: number) => void;
```

Reply:
After refactor this was removed